templates/main.php

```diff
@@ -36,7 +36,8 @@
 			</div>
 			<div id="app-settings-content">
 				<addressBookList></addressBookList>
-				<input type="file" id="contact-import" class="hidden-visually" contactimport /><label for="contact-import">Import</label>
+				<input type="file" id="contact-import" class="hidden-visually" contactimport />
+				<label for="contact-import" class="button" id="upload">Import<span class="icon-upload"></span></label>
```

Comment:
You can the commit which changes this to this branch and it will reappear in this pr